raiden-ts/src/helpers.ts

```diff
@@ -32,9 +32,9 @@ import ropstenServicesDeploy from './deployment/deployment_services_ropsten.json
 import rinkebyServicesDeploy from './deployment/deployment_services_rinkeby.json';
 import goerliServicesDeploy from './deployment/deployment_services_goerli.json';
 import mainnetServicesDeploy from './deployment/deployment_services_mainnet.json';
-import { UserDepositFactory } from './contracts/UserDepositFactory';
-import { MonitoringServiceFactory } from './contracts/MonitoringServiceFactory';
-import { TokenNetworkRegistryFactory } from './contracts/TokenNetworkRegistryFactory';
+import { UserDeposit__factory } from './contracts/factories/UserDeposit__factory';
```

Comment:
We would be better importing everything from `contracts` directly (since it's exported from `index.ts` instead of hardcoding these paths

Reply:
Also, if possible, where the contracts are used only for its types, use `import type`, if you can make this change on this PR as well

Reply:
✔️

Reply:
@palango ping